tests/contrib/httplib/test_httplib.py

```diff
@@ -165,7 +165,7 @@ def test_httplib_request_get_request_https(self):
                 we return the original response
                 we capture a span for the request
         """
-        conn = self.get_https_connection('httpbin.org')
+        conn = self.get_https_connection('httpbin-org.herokuapp.com')
```

Comment:
I guess this change must go in `master` too. Split the PR because we need it in both places.

Reply:
Alright, I'll split this out to a separate PR.